libr/socket/rap_server.c

```diff
@@ -4,6 +4,19 @@
 #include <string.h>
 #include <r_util.h>
 
+R_API RSocketRapServer *r_socket_rap_server_from_rsocket (int is_ssl, const char *port, RSocket *rap_sock) {
+	RSocketRapServer *rap_s;
+	if (!port)
+		return NULL;
+	rap_s = R_NEW0 (RSocketRapServer);
+	rap_s->fd = rap_sock;
+	memcpy (rap_s->port, port, 4);
```

Comment:
wat. i know condret wrote this crappy line. but it's better to fix it than expanding the bug all the way